### PR TITLE
Feature/logging service

### PR DESCRIPTION
**Pull request by Mautjee**
This PR introduces an logger function which only logs if environment is development.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -18,3 +18,22 @@ export const proofsToJSON = (
 ): JsonProof[] => {
   return proofs.filter((p) => !!p).map((p) => p?.toJSON()) as JsonProof[];
 };
+
+const isDevEnv = () => {
+  return process.env.NODE_ENV === "development";
```
Comment (Mautjee) [inline src/utils.ts:23]: Is this the right way of getting the environment variable? @silimarius

Comment (silimarius) [reply src/utils.ts:23]: Yes, just add a TODO to also make it work with import.meta.ENV in the bundler

**Review by silimarius — APPROVED** [review]
Dicaprio LGTM

File: src/zkAppWorker.ts [hunk 2]
```diff
@@ -51,31 +52,31 @@ setInterval(() => {
       });
     })
     .catch((err) => {
-      console.warn("Update queue error:", err);
+      logger.warn("Update queue error:", err);
```
Comment (ecovirtual) [inline src/zkAppWorker.ts:55]: Shouldn't this be `.error()` ?

**Review by ecovirtual — APPROVED** [review]

File: src/zkAppWorker.ts (hunk 2 above)
Comment (Mautjee) [reply src/zkAppWorker.ts:55]: @silimarius this was already a ``` console.warn ``` do you want to keep it an warn or make it an error?

**Pull request merged**